Signal/src/ViewControllers/ContactsViewHelper.h

```diff
@@ -0,0 +1,63 @@
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class ContactsViewHelper;
+@class Contact;
+@class ContactAccount;
+
+@protocol ContactsViewHelperDelegate <NSObject>
+
+- (void)contactsViewHelperDidUpdateContacts;
+
+- (BOOL)shouldHideLocalNumber;
+
+@end
+
+#pragma mark -
+
+@class OWSContactsManager;
+@class OWSBlockingManager;
+
+@interface ContactsViewHelper : NSObject
```

Comment:
This helper reduces the size of contact-related views by almost half.  It could be a shared singleton, but I haven't bothered making it one.